packages/three-vrm-materials-mtoon/src/MToonMaterialLoaderPlugin.ts

```diff
@@ -146,7 +146,7 @@ export class MToonMaterialLoaderPlugin implements GLTFLoaderPlugin {
     });
   }
 
-  private _getMToonExtension(materialIndex: number): V1MToonSchema.VRMCMaterialsMToon | undefined {
+  protected _getMToonExtension(materialIndex: number): V1MToonSchema.VRMCMaterialsMToon | undefined {
```

Comment:
[memo] It is necessary to be referenced from MToonNodeMaterialLoaderPlugin.getMaterialType()